src/tribler/gui/code_executor.py

```diff
@@ -60,7 +71,7 @@ def run_code(self, code, task_id):
             pass
 
         if self.shell.last_traceback:
-            self.on_crash(self.shell.last_traceback)
+            self.on_crash(self.shell.last_traceback + f'\n\ntask_id: {task_id!r}\ncode:\n{code}\n\n(end of code)')
```

Comment:
**nit** maybe this version is more readable:
```suggestion
            self.on_crash(f'{self.shell.last_traceback}\n\ntask_id: {task_id!r}\ncode:\n{code}\n\n(end of code)')
```

Reply:
Indeed!